trustymail/trustymail.py

```diff
@@ -13,14 +15,17 @@
 
 CSV_HEADERS = [
     "Domain", "Base Domain", "Live",
-    "MX Record", "Mail Servers",
+    "MX Record", "Mail Servers", "Sends Mail", "Supports STARTTLS",
     "SPF Record", "Valid SPF", "SPF Results",
     "DMARC Record", "Valid DMARC", "DMARC Results",
     "DMARC Record on Base Domain", "Valid DMARC Record on Base Domain",
     "DMARC Results on Base Domain", "DMARC Policy",
     "Syntax Errors"
 ]
 
+# The ports that will be checked to see if an SMTP server is listening
+SMTP_PORTS = [25, 465, 587]
```

Comment:
I think we need data from a full run on port 465. I'm of a mind not to include in our ports definition unless deployment is really strong in the federal space.

Reply:
I will leave this as it is for now, and we can look at the results to determine if 465 should be removed or not.  I will also create a flag to override the default ports with a comma-separated list of ports.